docs/Server-configuration.md

```diff
@@ -93,3 +93,66 @@ IMPOSTOR_Server__ListenPort=22023
 IMPOSTOR_AntiCheat__Enabled=true
 IMPOSTOR_AntiCheat__BanIpFromGame=true
 ```
+
+# systemd
+
+Linux users will likely want to create a systemd service definition to manage running the Impostor application.  Benefits to using a systemd service include the following:
+
+- The server process can automatically restart if the process dies
+- Impostor can be set to start on boot (handy if you do system updates regularly)
+- You can use systemd targets to ensure the process doesn't start until the system dependencies are met
+
+Using your favorite text editor, create the following service file:
+
+```
+sudo vim /etc/systemd/system/impostor.service
+```
+
+Modify the following template to your needs and populate the service file.
+
+```
+[Unit]
+Description=Impostor private Among Us server - https://github.com/Impostor/Impostor
+
+[Service]
+# Directory where Impostor is installed
+WorkingDirectory=/opt/impostor
+# Path to Impostor binary
+ExecStart=/opt/impostor/Impostor.Server
+Restart=always
+# Restart service after 10 seconds if the dotnet service crashes
+RestartSec=10
+KillSignal=SIGINT
+SyslogIdentifier=log.impostor
```

Comment:
Hmm, what is the effect of this line? Can you share an example log line?

For example, this is a log entry on my server:
```
Jan 19 23:15:24 euhtzfsn1 Impostor.Server[11330]: [23:15:24 WRN] Received RpcFlag for unregistered NetId 4.
```

Reply:
> Jan 20 02:25:39 amongus log.impostor[6741]: [02:25:39 INF] Client 3 disconnecting, reason: The remote sent a disconnect request

Reply:
ah okay, it just changes the name then. Not sure why we'd need the `log.` prefix then, maybe remove that prefix?